pkg/sql/plan2/function/builtin/multi/lpad.go

```diff
@@ -40,8 +59,11 @@ func Lpad(vecs []*vector.Vector, proc *process.Process) (*vector.Vector, error)
 	sourceStr := vector.MustBytesCols(vecs[ParameterSourceString]) //Get the first arg
 
 	//characters length not bytes length
-	lengthsOfChars := vector.MustTCols[int64](vecs[ParameterLengths])
-
+	// lengthsOfChars := vector.MustTCols[int64](vecs[ParameterLengths])
+	lengthsOfChars, err_ := Convert(vecs[ParameterLengths].Col)
```

Comment:
you can't do type conversion here.

just put the convert logic at function register at plan2/function/builtins.go.